bloom/util.py

```diff
@@ -287,6 +288,11 @@ def get_git_clone_state():
     return _disable_git_clone
 
 
+def get_distro_list_prompt():
+    global _distro_list_prompt
+    return _distro_list_prompt
```

Comment:
I'd prefer something like this:

```python
_distro_list_prompt = [
    'hydro',
    'indigo',
    'jade',
]

...

def get_distro_list_prompt():
    global _distro_list_prompt
    return ', '.join(_distro_list_prompt)
```

This will allow for cleaner diffs in the future when we add/remove distros.